### ISSUE TITLE
App cannot be opened

### ISSUE THREAD

**Issue by LorenzCoder** [iamrasel/lunar-launcher#111]
### Checklist

- [x] I made sure that there are **no existing issues** - [open](https://github.com/iamrasel/lunar-launcher/issues) or [closed](https://github.com/iamrasel/lunar-launcher/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### App version

2.8.1.1

### Android version

Android 13

### Steps to reproduce the bug

When I want to open the app, it kills itself immediately

### Expected behavior

Set the app as a launcher

**Comment by clicky6** [comment]
> ### Checklist
> 
>     * [x]  I made sure that there are **no existing issues** - [open](https://github.com/iamrasel/lunar-launcher/issues) or [closed](https://github.com/iamrasel/lunar-launcher/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
> 
>     * [x]  I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
> 
>     * [x]  This issue contains only one bug.
> 
> 
> ### App version
> 
> 2.8.1.1
> ### Android version
> 
> Android 13
> ### Steps to reproduce the bug
> 
> When I want to open the app, it kills itself immediately
> ### Expected behavior
> 
> Set the app as a launcher
> ### Screenshots/Screen recordings
> 
> _No response_
> ### Logs
> 
> _No response_
> ### Additional information
> 
> _No response_

+1
Same Issue

**Comment by OneRadRedDad** [comment]
Same here. Galaxy S10e. Android 12.

**Comment by LorenzCoder** [comment]
> Same here. Galaxy S10e. Android 12.

@OneRadRedDad Have you tried it again now?

**Comment by iamrasel** [comment]
Please try the latest release and inform me if it has been solved. @LorenzCoder @clicky6 @OneRadRedDad

**Comment by OneRadRedDad** [comment]
Yes, it works for me now!

It's worth noting I made my comment based on the version installed from F-Droid. This is listed as version 2.8.1.1, with a release date of 1/15/24.

After seeing your comment, @iamrasel, I downloaded the latest version from github. Android thought the github version was an older version than the one from F-Droid and so refused to "upgrade" the installation. I had to uninstall the F-Droid version to then install the latest github release.
